pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -64,7 +64,7 @@ workflow Optimus {
 
   # version of this pipeline
 
-  String pipeline_version = "6.1.0"
+  String pipeline_version = "6.2.0"
```

Comment:
should this be a minor patch if it doesnt actually change the optimus outputs?

Reply:
I'll change to minor patch